### PR TITLE
Feature/plotting

### PR DESCRIPTION
**Pull request by martinfalk**
* new `AxisStyleProperty` to adjust multiple axis properties at the same time (line/text color, font size, tick length). 
* new processor `ImagePlot` for plotting a 2D image with matching x-y axes. Interaction events are propagated, i.e. it is possible to browse through volume slices (see example)

image

### CODE REVIEW

File: modules/plotting/src/properties/axisstyleproperty.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        // use the file name w/o extension as identifier
+        fontFace_.addOption(name, font.first, font.second);
+    }
+    fontFace_.setSelectedIdentifier("Montserrat-Medium");
+    fontFace_.setCurrentStateAsDefault();
+    fontSize_.setSemantics(PropertySemantics("Fontsize"));
+    color_.setSemantics(PropertySemantics::Color);
+
+    fontFace_.onChange([this]() {
+        for (auto a : axes_) {
```
Comment (ResearchDaniel) [inline modules/plotting/src/properties/axisstyleproperty.cpp:69]: might want to introduce a network lock for these?

**Review by ResearchDaniel — APPROVED** [review]
Sweet!!!
I can imagine that having image(s) in the regular plot should be possible too, basically the merging of ImagePlotProcessor and scatter plot. Would it be possible to reuse this code to do something like that?

**Review by petersteneteg — COMMENTED** [review]
Some comments

**Review by petersteneteg — APPROVED** [review]

File: modules/plotting/src/properties/axisstyleproperty.cpp (hunk 1 above)
Comment (martinfalk) [reply modules/plotting/src/properties/axisstyleproperty.cpp:69]: done

**Comment by martinfalk** [comment]
rebased into #599

**Pull request closed without merging**